README.md

```diff
@@ -48,6 +48,17 @@ npx pod-install
 1. Create a new app in firebase with the package with the suffix of the flavor you will run
 2. Download the `google-services.json` and move it to the `./android/app` directory
 
+## EAS Build
+
```

Comment:
WDYT about adding the steps to install eas locally can be added to the readme with the link to the official doc?

Reply:
Done